From f92845e4c93da43feb52261916a6a979eff7bd0d Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Thu, 20 Jun 2019 17:20:13 -0400
Subject: [PATCH] NOT UPSTREAM: Build fix with latest kernel

From here: https://lkml.org/lkml/2019/6/3/988

Some attempts at patches have been posted, check qemu thread logs
for SIOCGSTAMP

Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 linux-user/syscall.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux-user/syscall.c b/linux-user/syscall.c
index 39a37496fe..830dc876ed 100644
--- a/linux-user/syscall.c
+++ b/linux-user/syscall.c
@@ -34,6 +34,7 @@
 #include <sys/resource.h>
 #include <sys/swap.h>
 #include <linux/capability.h>
+#include <linux/sockios.h>
 #include <sched.h>
 #include <sys/timex.h>
 #include <sys/socket.h>
